testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -69,3 +72,98 @@ def test_optimize_cutoff(universe, lipid_heads):
     cutoff, N = optimize_cutoff(universe, lipid_heads, pbc=True)
     assert N == 2
     assert_almost_equal(cutoff, 10.5, decimal=4)
+
```

Comment:
PEP8 change: you should have two blank lines between functions, same thing applies for functions further down.